config/locales/idv/en.yml

```diff
@@ -132,21 +132,27 @@ en:
       sessions:
         no_pii: Do not use real personal information (demo purposes only)
     modal:
+      attempts:
+        one: 1 attempt
+        other: "%{count} attempts"
       button:
         warning: Try again
       finance:
         warning_accent: We could not find records matching your financial information.
         warning_html: >
-          %{accent} <span>Please check the information you entered.</span>
+          %{accent} <span>Please check the information you entered.</span> <span>
```

Comment:
thoughts on having `warning_header_html` and `warning_body_html` for each of these so we can put the `span` tags in the controller?